src-executables/Main.hs

```diff
@@ -103,6 +104,7 @@ fstatsOptParser = FstatsOptions <$> parseBasePaths
                                 <*> OP.many parseStatSpecsDirect
                                 <*> parseStatSpecsFromFile
                                 <*> parseRawOutput
+                                <*> parseIgnoreChecksums
```

Comment:
Hmm. Ist dieses Opt-out des Checksum-checks die beste Lösung? Sollten wir nicht einfach Warnings ausgeben wenn die Checksums nicht passen? Das sollte doch in der Regel schnell zu beheben sein, oder?